.travis.yml

```diff
@@ -1,21 +1,36 @@
+sudo: false
 language: python
 
-python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "pypy-5.3.1"
+cache:
+  directories:
+    - $HOME/.cache/pip
```

Comment:
While we’re caching stuff, let’s cache `.hypothesis` as well. That ensures the examples database is preserved over consecutive runs.